Modifying Website Sale Cart View

TimLai125 opened the pull request.

TimLai125 committed:
Modifying Website Sale Cart View

yostashiro commented on website_sale_adj/__manifest__.py:
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Insert this:
```
# Copyright 2017 Quartile Limited
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

yostashiro commented on website_sale_adj/__manifest__.py:
```diff
@@ -0,0 +1,20 @@
+* Adjust shopping cart view
+    """,
+    'author': 'Quartile Limited',
+    'license': 'AGPL-3',
```
`LGPL-3`

yostashiro commented on website_sale_adj/views/templates.xml:
```diff
@@ -0,0 +1,19 @@
+                    </a>
+                <div class="oe_structure" />
+            </div>
+        </xpath>
```
Replace above xpath with:
```
        <xpath expr="//div[@class='col-md-8 col-sm-9 oe_cart']" position="attributes">
            <attribute name="class">col-md-12 col-sm-12 oe_cart</attribute>>
        </xpath>
```

yostashiro reviewed (changes requested):
Can we rename the module to `website_sale_view_adj`?

yostashiro commented:
![image](https://cloud.githubusercontent.com/assets/7766939/26460334/f2e58fb2-41ab-11e7-8570-fde7a82b55b3.png)

Please also remove this from the product page.

TimLai125 committed:
Renaming Module website_sale_adj to website_sale_view_adj
Correcting the license type and adding comments
Improving code sytnax
Modifiying on product's view
